www/include/configuration/configObject/contactgroup/DB-Func.php

```diff
@@ -190,26 +194,36 @@ function insertContactGroup($ret)
         $ret = $form->getSubmitValues();
     }
 
-    $ret["cg_name"] = $centreon->checkIllegalChar($ret["cg_name"]);
+    $cgName = $centreon->checkIllegalChar(filter_var($ret["cg_name"], FILTER_SANITIZE_STRING));
+    $cgAlias = filter_var($ret["cg_alias"], FILTER_SANITIZE_STRING);
+    $cgComment = filter_var($ret["cg_comment"], FILTER_SANITIZE_STRING);
+    $cgActivate = filter_var($ret["cg_activate"]["cg_activate"], FILTER_SANITIZE_STRING); //enum
```

Comment:
this line seems obsolete, as the enum will returns the state of the checkbox, which is checked on next line